src/main/java/oracle/TaskList.java

```diff
@@ -0,0 +1,89 @@
+package oracle;
+
+import command.CommandFormatException;
+import entry.Deadline;
+import entry.Event;
+import entry.Task;
+import entry.Todo;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class TaskList {
+    private final ArrayList<Task> tasklist;
+
+    /** Creates a new TaskList, which stores the tasks. This class provides some abstraction for the underlying
+     * ArrayList<Task
+     * @param load given the raw strings from the storage file, we create the tasks
+     */
+    public TaskList(List<String> load) {
+        this.tasklist = new ArrayList<>();
+        for (String s : load) {
+            String[] sorted = s.split("\u001E");
+            Boolean isDone = sorted[1].equals("T");
+            try{
+                switch (sorted[0]) {
```

Comment:
Add a default case as specific in textbook.

Reply:
yup thanks